force_wfmanager/wfmanager_setup_task.py

```diff
@@ -206,30 +209,25 @@ def _tool_bars_default(self):
                     method="open_plugins",
                     image_size=(64, 64)
                 ),
-            ),
-            SToolBar(
-                TaskAction(
-                    name="Run",
-                    tooltip="Run Workflow",
-                    image=ImageResource("baseline_play_arrow_black_48dp"),
-                    method="run_bdss",
-                    enabled_name="run_enabled",
-                    image_size=(64, 64)
-                ),
+
             )
         ]
 
     def _default_layout_default(self):
         """ Defines the default layout of the task window """
         return TaskLayout(
-            left=PaneItem('force_wfmanager.tree_pane'),
+            left=PaneItem('force_wfmanager.side_pane')
         )
 
+    # Overloaded Methods
+
     def create_central_pane(self):
         """ Creates the central pane which contains the layer info part
         (factory selection and new object configuration editors)
         """
-        return SetupPane(workflow_model=self.workflow_model)
+        return SetupPane(
```

Comment:
The `SetupPane` now only listens to and displays objects in `SystemState`. It does not hold any information about the `Workflow`